### ISSUE TITLE
As a user, I want to see bibliographic information (title, publication date, publisher, link to OCLC if available) for an individual library item, so that I can more accurately identify the item.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/mep-django#234]
# testing notes
## designs
[desktop](https://app.zeplin.io/project/5c48e7313d2b13bf7fc9fe14/screen/5e5d1a05af422b653b9fc8ff) / [tablet](https://app.zeplin.io/project/5c48e7313d2b13bf7fc9fe14/screen/5e5d1a597330b461200037d9) / [mobile](https://app.zeplin.io/project/5c48e7313d2b13bf7fc9fe14/screen/5e5d1a794085d175a8592307)
## checklist
this issue has a variety of different things to test (including on the admin side). in order to support the display of a work's creators on the detail page, we added an "order" property to creator types, so that you can choose the order in which creators will be displayed (e.g. make sure authors are listed first, followed by editors, etc).

you will want to have a variety of works to test with; ideally at least one of each of:
- a periodical with issue info ("The Dial")
- a book with a "read online" link ("The Garden Party")
- a book with multiple authors ("The English Novelists")
- a book with multiple kinds of creators ("Capajon")
- a book with public notes ("Dante")

visit each of these detail pages, and check that:
1. all of the creators are listed and match the design
2. the work has the correct format indicator (book, periodical)
3. the "read online" link works
4. the public notes are displayed
5. the list of issues is displayed

note that the uncertainty indicator is not yet visible, since that's #524.

after you test this out, head to the admin backend and look at the creator types.
1. you should see a new property of each creator type, "order"
2. some of the existing creator types should automatically have an order assigned to them (for example, "author" should be 1, so that it comes first)
3. try changing the order values around (for example, put editors first) and visit a detail page. did the order of creators change?
4. make a new creator type (anything will do; I tested with "binder"). it should automatically receive the order value 20.

**Comment by thatbudakguy** [comment]
bumping to a 5 because this required model changes and there were no view tests.

**Comment by clmahoney** [comment]
Desktop: 
- YES, I checked periodical "transition," saw everything I should be seeing (all of the creators are listed and match the design; the work has the correct format indicator (book, periodical); the list of issues is displayed) 
![Screen Shot 2020-04-03 at 12 46 19 PM](https://user-images.githubusercontent.com/16673934/78385043-3954c100-75a9-11ea-9e8f-e6ca3deeb771.png)

- NO, when I checked "The Garden Party" I did not see a link for reading online: 
![Screen Shot 2020-04-03 at 12 48 37 PM](https://user-images.githubusercontent.com/16673934/78385228-8769c480-75a9-11ea-8bae-7b459ed1c75d.png)
--> I also checked "The Psychology of Sex" and saw no online link

- YES, when I checked the multiple author book "The English Novelists," I DID see the multiple authors but I did NOT see the differentiation of "Book": 
![Screen Shot 2020-04-03 at 12 51 12 PM](https://user-images.githubusercontent.com/16673934/78385417-d0ba1400-75a9-11ea-851e-4b462f76ae99.png)

- YES, when I checked the multiple creator "Capajon," I DID see the multiple creators but did NOT see the "Book" box: 
![Screen Shot 2020-04-03 at 12 52 14 PM](https://user-images.githubusercontent.com/16673934/78385511-f810e100-75a9-11ea-85ae-595c13165249.png)

- YES, a book with a public note shows the public note. I checked "Dante," and I had put in a test public note: 
![Screen Shot 2020-04-03 at 12 54 35 PM](https://user-images.githubusercontent.com/16673934/78385709-4e7e1f80-75aa-11ea-9b69-5e8120787cbf.png)

**Comment by clmahoney** [comment]
Mobile: 
- YES, periodical shows issues and "Periodical" format (tried "transition" and "New Masses"
- YES, a book with a "read online" link works - I tried "The Garden Party"
- YES, a book with multiple authors show all authors. I tried "A History of Sculpture."
- YES, a book with multiple kinds of creators shows all creators. I tried "Capajon," and saw the multiple authors, the introduction writer, and the translator. 
- YES, a book with public notes shows the public note. I checked "Dante" again. 
- YES,

**Comment by clmahoney** [comment]
Tablet: 
- All looks good here too!

**Comment by clmahoney** [comment]
Admin backend of Creator Types: 
- YES, I see the "Order" of the creator types: 
![Screen Shot 2020-04-03 at 1 32 53 PM](https://user-images.githubusercontent.com/16673934/78388808-a4a19180-75af-11ea-819c-7427536340ae.png)
- YES, I see the designated order (see above screenshot)
- YES, when I changed the order and then went to a detail page, the order that I created showed up. I made Introduction #1, and that was how it was in the book detail page. For some reason thought it took forever for the page to save and load again when I changed the order. But then I did a hard refresh and it was faster!
- NO, when I tried to add a new creator type, it wouldn't save without me manually entering in an order. Then when I entered an order # of 7, it ordered it alphabetically: 
![Screen Shot 2020-04-03 at 1 39 16 PM](https://user-images.githubusercontent.com/16673934/78389341-91db8c80-75b0-11ea-80ed-ab4e541fb9b0.png)

**Comment by thatbudakguy** [comment]
@clmahoney I actually wrote the notes wrong on this - the behavior you saw is the correct one! sorry about that.

what should happen is:
- when you create a new creator type, you _have_ to specify its order (but any number is fine; duplicates are OK)
- if there were any "extra" creator types existing before this update (i.e. beyond author, editor, etc) they would get a default value of 20 until you changed them. but it appears we anticipated all the existing types, so this didn't happen. which is fine!

the alphabetic ordering is because "name" is chosen for ordering by default on this page; if you click the "order" column it will order by the order values instead. if it makes more sense to order them here the same way we order them on the book detail page, I think that would be an easy change to make.
